tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php

```diff
@@ -120,6 +121,18 @@ public function testAssertMatchesJsonSchema(): void
         $this->assertMatchesJsonSchema(json_decode($jsonSchema, true));
     }
 
+    public function testAssertMatchesResourceCollectionJsonSchema(): void
+    {
+        self::createClient()->request('GET', '/resource_interfaces');
+        $this->assertMatchesResourceCollectionJsonSchema(ResourceInterface::class);
```

Comment:
This test is failing because of https://github.com/justinrainbow/json-schema/issues/523

I'm not sure what we could do here.